docs/resources/virtual_server.md

```diff
@@ -4,6 +4,8 @@ page_title: "anxcloud_virtual_server Resource - terraform-provider-anxcloud"
 subcategory: ""
 description: |-
   The virtual_server resource allows you to configure and run virtual machines.
+  Known limitations
+  removal of disks not supportedremoval of networks not supported
```

Comment:
Missing space

Reply:
Unfortunately the docs generator doesn't handle this very well.. I'll remove those lines manually from the description in this file as they don't have any value here.